core/App/ehsm_provider.cpp

```diff
@@ -1013,6 +1044,15 @@ ehsm_status_t VerifyQuote(ehsm_data_t *quote,
     //set current time. This is only for sample use, please use trusted time in product.
     current_time = time(NULL);
 
+    // get mrenclave and mrsigner from quote data
+    get_mrenclave_mrsigner_from_quote(quote->data, mrsigner_quote, mrenclave_quote);
```

Comment:
rename the function to bool verify_policy(quote, mrsigner_good, mrenclave_good), and check the return value.

so you can remove the line#984-#987

Reply:
copy understood